assets/src/js/orders.js

```diff
@@ -332,10 +332,12 @@ jQuery(function($) {
 
         refunds: {
 
-            do_refund: function() {
+            do_refund: async function() {
                 dokan_seller_meta_boxes_order_items.block();
+                
+                let isRefund = await dokan_sweet_alert( dokan_refund.i18n_do_refund, { action : 'confirm' }  );
```

Comment:
if the variable used only once and not being updated, use `const` instead of `let`